lib/phonebook/ui/copiabled_text.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:myecl/phonebook/tools/constants.dart';
+import 'package:myecl/tools/functions.dart';
+
+class CopiabledText extends StatelessWidget {
+  const CopiabledText(
+      {Key? key,
+      required this.data,
```

Comment:
Since it is an extension of Text, it could be cool to mimic its behavior, regarding the first parameter.
You can take this argument out of the braces to prevent passing its name.

`CopiabledText(this.data, {Key? key,`